### ISSUE TITLE
Thank you for what you're doing

### ISSUE THREAD

**Issue by jrosebr1** [ieee8023/covid-chestxray-dataset#20]
Hi there, @ieee8023. I just wanted to take a second and thank you for what you're doing.

I run a popular computer vision/deep learning blog (PyImageSearch.com) and published a tutorial for CV/DL practitioners on how to use this dataset (your work and dataset curation is properly cited):

https://www.pyimagesearch.com/2020/03/16/detecting-covid-19-in-x-ray-images-with-keras-tensorflow-and-deep-learning/

I'll be honest -- it's one of the least scientific things I've published (but that's not the point of the piece).

It's mainly just for people (including myself) to "feel" like they are helping and be able to take solace in it (and educate themselves via it). It's an opportunity for people to be inspired by it. And while it won't save any lives immediately, perhaps downstream, it will help people apply themselves and learn a new skill while they are quarantined or displaced from the work, school, or research lab.

I don't know if it's worthy of being included in your repo, but I wanted to pass it along just in case.

Thank you again, you're doing amazing work.

**Comment by ieee8023** [comment]
> It's mainly just for people (including myself) to "feel" like they are helping and be able to take solace in it (and educate themselves via it)

I am not trying to educate people. I am trying to build a dataset to build tools that are useful. 

I find the main issue with your work is the evaluation. You simply cannot claim the model has such high performance. You are only using train and valid sets and no external test set. You should also perform some sort of cross-validation to assess how well the model generalizes.

**Comment by jrosebr1** [comment]
Fair criticism and honest criticism, I appreciate it. As I stated in the work it was never intended to be a true scientific piece with a proper evaluation. My intention was simply to share it with you, that's all.

I wish you all the best with the project, good luck with it.
